ddl/scheduler.go

```diff
@@ -281,6 +291,14 @@ func (b *backfillSchedulerHandle) CleanupSubtaskExecEnv(context.Context) error {
 
 // Rollback implements the Scheduler interface.
 func (b *backfillSchedulerHandle) Rollback(context.Context) error {
+	logutil.BgLogger().Info("[ddl] rollback backfill add index task")
+	bc, err := ingest.LitBackCtxMgr.Register(b.d.ctx, b.index.Unique, b.job.ID)
```

Comment:
Why `ingest.LitBackCtxMgr.Register` should be called here?

Reply:
+1. I suggest to use `ingest.LitBackCtxMgr.Load()` instead.